components/Tabs/Tabs.js

```diff
@@ -1,24 +1,27 @@
-
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
```

Comment:
element.classList is a read-only property that contains the list of class attributes that an element, or node, contains.  
You used classList.remove() below.  The same thing can be done here.
```
const item = this.element.classList.add('[insert name of class to add here]')
```